test/ConcreteOperations/minkowski_sum.jl

```diff
@@ -0,0 +1,8 @@
+for N in [Float64, Float32, Rational{Int}]
+    X = Interval(N(1), N(2))
+    Y = X + X
+    if test_suite_polyhedra
+        minkowski_sum(X, Y)
```

Comment:
did you intend want to `@test` something here?

Reply:
No, just that it does not crash. I do not have a good `@test` in mind.